src/mngt/ocf_mngt_cache.c

```diff
@@ -2050,7 +2064,7 @@ static void _ocf_mngt_cache_unplug(ocf_cache_t cache, bool stop,
 	__deinit_cleaning_policy(cache);
 	__deinit_promotion_policy(cache);
 
-	if (ocf_mngt_cache_is_dirty(cache)) {
+	if (dirty) {
```

Comment:
Why can't we just do this earlier (before removing cores)?

Reply:
we can, but it seems to belong here logically - don't you think?

Reply:
Imo we could even do it as a first thing during cache stop. We don't expect any flushing at that point anyway.